api/src/identity-access-management/domain/usecases/add-oidc-provider.js

```diff
@@ -74,32 +83,14 @@ const addOidcProvider = async function ({
     slug,
     source,
     isVisible,
-  });
+  };
+  addOidcProviderValidator.validate(properties);
 
   const encryptedClientSecret = await cryptoService.encrypt(clientSecret);
+  const propertiesWithoutClientSecret = (({ clientSecret: _, ...rest }) => rest)(properties);
```

Comment:
Ça me parait plus simple d'écrire de cette façon:

```suggestion
  const { clientSecret: _, ...propertiesWithoutClientSecret } = properties;
```

Reply:
D’accord, mais ça oblige à ajouter une exception eslint :
```javascript
// eslint-disable-next-line no-unused-vars
const { clientSecret: _, ...propertiesWithoutClientSecret } = properties;
```
sinon on a l’erreur : 
```
error  '_' is assigned a value but never used  no-unused-vars
```

Il semble qu’avec notre configuration ESLint `Allowed unused args must match /_/u` fonctionne sur les args, pas sur les destructurations. Note : Voir `destructuredArrayIgnorePattern`.

Reply:
J'avoue que je préfère l'ajout de l'exception eslint plutôt que d'avoir ce contournement de règle.

À voir ensemble si on ne peut pas améliorer les règles eslint.

Reply:
Parfait, tout à fait d’accord.

Reply:
Nous proposerons une amélioration des règles eslint dans une autre PR.

Reply:
La modification a été faite pour utiliser simplement la destructuration, avec une exception eslint.